docs/src/pages/getting-started/support/support.md

```diff
@@ -79,6 +79,7 @@ Accelerate your company’s application development with a secure, long-term sol
 - **Version migration assistance**
 - **Priority fixes**
 - **Support**. Timely support and troubleshooting when you need it most. Get expert help directly from the Material-UI team with guaranteed response times (next-business day).
+- **Access to premium modules**. Datagrid, Charts, Scheduler, Rich Text Editor, Splitter, and more.
```

Comment:
Doesn't exist, so shouldn't be offered...

Reply:
What about "Fake it until you make it“? cf https://marmelab.com/react-admin/

![Capture d’écran 2019-09-19 à 13 43 29](https://user-images.githubusercontent.com/3165635/65241299-7bbe2100-dae3-11e9-92f7-33046ac1e326.png)

Reply:
You might get away with that in B2C (depending on the context), but not in B2B.

Also, most of those are months (or more) of of effort. So even if you "make it", you're now on the hook to deliver them before contract start date.

Reply:
> What about "Fake it until you make it“?

This saying is not meant to be taken literally. It's used in the context of confidence about something or change in behavior. These components literally do not exist so it's not "fake it until you make it" but lying about the existence of something.

I'm pretty sure we can be immediately sued if someone buys that and there are no components. Even if we decide to make it a pre-order we're required to deliver it within a legally defined timeframe.

And even if you say "well they can't order it they have to contact first": How do you think most will react if they ask about those components and we don't have them? I don't think this will boost confidence in our product.

Reply:
Thanks for the feedback. I think that mentioning actual components is wrong. To be clear, the idea is to spark conversations with a bait so we can learn from people use cases. I don't think that we should try to sell something that doesn't exist yet, I also doubt a lot of companies will buy React components they can't use right away. 

I would propose a follow-up pull request where we only talk about "premium modules". I would also propose that we add a disclaimer in the roadmap.md that it's subject to change.

Would it be OK with you?

Reply:
I'd suggest removing all mention, until such time as there is something to offer.

Reply:
How would we know what to build, and what first?